pkg/controller/collection/collection_controller_test.go

```diff
@@ -337,60 +341,166 @@ func (ch collectionHandler) ServeHTTP(rw http.ResponseWriter, req *http.Request)
 }
 
 type fileInfo struct {
-	name string
+	name   string
 	sha256 string
 }
 
 const (
-	myuid = "MYUID"
+	myuid    = "MYUID"
 	otheruid = "OTHERUID"
 )
 
 var basicPipeline = fileInfo{
-	name: "/basic.pipeline.tar.gz",
+	name:   "/basic.pipeline.tar.gz",
 	sha256: "89277396f1f7083ae4c16633949f45860d164534212720b17f25e438e00f66af"}
 
 var badPipeline = fileInfo{
-	name: "/bad.pipeline.tar.gz",
+	name:   "/bad.pipeline.tar.gz",
 	sha256: "6e2d419d7971b2ea60148a995914acda4e42c89cc4b38513aeba0aee18c944f3"}
 
 var digest1Pipeline = fileInfo{
-	name: "/digest1.pipeline.tar.gz",
+	name:   "/digest1.pipeline.tar.gz",
 	sha256: "0238ff31f191396ca4bf5e0ebeea323d012d5dbc7e3f0997e1bf66b017228aaf"}
 
 var digest2Pipeline = fileInfo{
-	name: "/digest2.pipeline.tar.gz",
+	name:   "/digest2.pipeline.tar.gz",
 	sha256: "c3f28ffca707942a8b351000722f1aebda080e3706aa006650a29d10f4aa226b"}
 
 var triggerPipeline = fileInfo{
-	name: "/trigger.pipeline.tar.gz",
+	name:   "/trigger.pipeline.tar.gz",
 	sha256: "bba27927b6da1323f264e957c526c4df118d657dd8ad19e08994d70caaf31f73"}
 
+// --------------------------------------------------------------------------------------------------
+// Test collection/stack id validation.
+// --------------------------------------------------------------------------------------------------
+func TesCollectionIDValidation(t *testing.T) {
```

Comment:
Typo - Tes -> Test, these tests won't run if the method name is spelled this way.